x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/saml/SamlAuthnRequestBuilder.java

```diff
@@ -24,12 +28,14 @@
     private final String idpBinding;
     private Boolean forceAuthn;
     private NameIDPolicySettings nameIdSettings;
+    private List<String> reqAuthnCtxClassRef;
 
     SamlAuthnRequestBuilder(SpConfiguration spConfig, String spBinding, EntityDescriptor idpDescriptor, String idBinding, Clock clock) {
         super(idpDescriptor, spConfig, clock);
         this.spBinding = spBinding;
         this.idpBinding = idBinding;
         this.nameIdSettings = new NameIDPolicySettings(NameID.TRANSIENT, false, null);
+        this.reqAuthnCtxClassRef = spConfig.getReqAuthnCtxClassRef();
```

Comment:
You don't need to store this in a field, it's available as `super.serviceProvider.getReqAuthnCtxClassRef()`